src/app/new-courses-page/new-course-form/new-course-form.component.ts

```diff
@@ -17,7 +24,7 @@ export class NewCourseFormComponent implements OnInit {
   value: Partial<Course> = {};
   courseTitle: string = "New course";
 
-  ngOnInit(): void {
+  ngAfterContentChecked(): void {
```

Comment:
why do you use afterContent checked? This hook is intended to work with content children. You don't need it. Please reconsider your approach